lab_4_summarization_textrank/main.py

```diff
@@ -120,23 +170,42 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes
         """
-        pass
+        super().__init__()
+        self._word2id = {}
```

Comment:
this attribute will be created by the parent class initializer

Reply:
and the following one

Reply:
k, i thought maybe there was a need to overwrite them